lib/cloak/fields/pbkdf2.ex

```diff
@@ -0,0 +1,193 @@
+defmodule Cloak.Fields.PBKDF2 do
```

Comment:
This module should be wrapped in `if Code.ensure_loaded?(PBKDF2)`. If PBKDF2 is going to be an optional dependency, we need to ensure that the compiler won't attempt to compile this file when it isn't present.

Reply:
Wow, that's neat. I haven't seen that before. Thanks for teaching me about it!